From b791a75f68cd350a4e4a1b836c7214ebf9c20407 Mon Sep 17 00:00:00 2001
From: Pankaj Bharadiya <pankaj.laxminarayan.bharadiya@intel.com>
Date: Wed, 13 Dec 2017 15:51:28 +0530
Subject: [PATCH 0988/1569] Soundwire: fix soundwire slave status arrary size

Soundwire master handles SOUNDWIRE_MAX_DEVICES+1 slave addresses hence
set sdw_slave_status size to SOUNDWIRE_MAX_DEVICES+1.

Change-Id: I2cc82a084927a7241b2e03a7f3a8791269f660c3
Review-by: Vaibhav Agarwal <vaibhav.agarwal@intel.com>
Signed-off-by: Pankaj Bharadiya <pankaj.laxminarayan.bharadiya@intel.com>
---
 drivers/sdw/sdw_priv.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/sdw/sdw_priv.h b/drivers/sdw/sdw_priv.h
index b165d72..0a6d57d 100644
--- a/drivers/sdw/sdw_priv.h
+++ b/drivers/sdw/sdw_priv.h
@@ -199,7 +199,7 @@ struct sdw_runtime {
 
 struct sdw_slv_status {
 	struct list_head node;
-	enum sdw_slave_status status[SOUNDWIRE_MAX_DEVICES];
+	enum sdw_slave_status status[SOUNDWIRE_MAX_DEVICES+1];
 };
 
 /** Bus structure which handles bus related information */
-- 
2.7.4

